### DISCUSSION TITLE
Server Localized Description (in config.json)

### DISCUSSION THREAD

**Discussion by sunnamed434** [SmartlyDressedGames/Unturned-3.x-Community#3905] [Ideas]
The idea is to make able to localize everything as much as possible, for example, would be great to specify the description depending on the language of the SteamPlayer

**Comment by ItsRodrigoAl** [comment]
https://forum.smartlydressedgames.com/t/how-will-be-translations-implemented-and-maybe-any-eta-for-it/22748/3

i did posted something related with it on the forum, no reply yet

**Reply by sunnamed434** [reply]
interesting
